src/test/java/com/hotel/mappers/response/HotelRateRuleResponseMapperTest.java

```diff
@@ -0,0 +1,57 @@
+package com.hotel.mappers.response;
+
+import com.hotel.service.raterule.HotelRateRuleResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opentravel.ota._2003._05.*;
```

Comment:
please import specific classes instead of *

Reply:
done